.env.example

```diff
@@ -0,0 +1,52 @@
+APP_NAME=Escooters
+APP_ENV=local
+APP_KEY=
+APP_DEBUG=true
+APP_URL=http://escooters.blumilk.localhost
+
+LOG_CHANNEL=stack
+LOG_DEPRECATIONS_CHANNEL=null
+LOG_LEVEL=debug
+
+DB_CONNECTION=pgsql
+DB_HOST=escooters-db-dev
+DB_PORT=5432
+DB_DATABASE=escooters
+DB_USERNAME=escooters
+DB_PASSWORD=password
+
+BROADCAST_DRIVER=log
+CACHE_DRIVER=file
+FILESYSTEM_DISK=local
+QUEUE_CONNECTION=sync
+SESSION_DRIVER=file
+SESSION_LIFETIME=120
+
+MEMCACHED_HOST=127.0.0.1
+
+REDIS_HOST=127.0.0.1
+REDIS_PASSWORD=null
+REDIS_PORT=6379
+
+MAIL_MAILER=smtp
+MAIL_HOST=mailhog
+MAIL_PORT=1025
+MAIL_USERNAME=null
+MAIL_PASSWORD=null
+MAIL_ENCRYPTION=null
+MAIL_FROM_ADDRESS="hello@example.com"
+MAIL_FROM_NAME="${APP_NAME}"
+
+CURRENT_UID=1000
+DB_ROOT_PASSWORD=example
+
+# DOCKER
+DOCKER_APP_HOST_PORT=3851
+DOCKER_INSTALL_XDEBUG=true
+DOCKER_DATABASE_HOST_PORT=3853
+DOCKER_MAILPIT_DASHBOARD_HOST_PORT=3856
+DOCKER_REDIS_HOST_PORT=3852
+
+DOCKER_HOST_USER_ID=1000
+
+XDG_CONFIG_HOME=/tmp
```

Comment:
For what is it needed?

Reply:
I added that because tinker has problem sometimes without this but now we can remove that.

Reply:
removed